### PR TITLE
chore(e2e): dont check for immich folder

### PR DESCRIPTION
**Pull request by etnoy**
I don't know why, but sometimes my e2e tests fail due to the .immich folder missing. I think it makes sense to ignore that check when running e2e

### CODE REVIEW

File: e2e/docker-compose.yml [hunk 1]
```diff
@@ -32,7 +33,7 @@ services:
       - 2285:3001
 
   redis:
-    image: redis:6.2-alpine@sha256:2ba50e1ac3a0ea17b736ce9db2b0a9f6f8b85d4c27d5f5accc6a416d8f42c6d5
+    image: redis:6.2-alpine@sha256:2d1463258f2764328496376f5d965f20c6a67f66ea2b06dc42af351f75248792
```
Comment (jrasm91) [inline e2e/docker-compose.yml:36]: Can you revert this change?

**Review by jrasm91 — APPROVED** [review]

File: e2e/docker-compose.yml (hunk 1 above)
Comment (etnoy) [reply e2e/docker-compose.yml:36]: Done. No idea how it happened!

**Review by danieldietzler — APPROVED** [review]

**Comment by jrasm91** [comment]
Looks like you have conflicts to resolve now.

**Comment by etnoy** [comment]
> Looks like you have conflicts to resolve now.

Fixed now

**Review by alextran1502 — APPROVED** [review]
